pmm/aws-staging-start.groovy

```diff
@@ -304,9 +304,20 @@ pipeline {
                         sudo yum -y update --security
                         sudo yum -y install https://repo.percona.com/yum/percona-release-0.1-7.noarch.rpm
                         sudo rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY
-                        sudo yum -y install svn docker sysbench mysql57-server git php php-mysql php-pdo
+                        sudo yum -y install https://dev.mysql.com/get/mysql57-community-release-el7-11.noarch.rpm 
+                        sudo yum -y install git gcc make automake libtool openssl-devel ncurses-compat-libs
+                        sudo yum -y install svn docker git php php-mysql php-pdo mysql-community-devel mysql-community-client mysql-community-common mysql-community-server
+                        git clone https://github.com/akopytov/sysbench
+                        cd sysbench
+                        ./autogen.sh
+                        ./configure
+                        make
+                        sudo make install
+                        sysbench --version
+                        cd ..
                         sudo service mysqld start
-                        sudo yum -y install bats --enablerepo=epel
+                        sudo amazon-linux-extras install epel -y
```

Comment:
By default after this command epel repo is installed but not enabled so for line 320 it is still recommended to use --enablerepo=epel

Reply:
it seems the earlier command installed epel and enabled it also I tested it practically